src/scss/footer.scss

```diff
@@ -0,0 +1,121 @@
+footer {
+	width: calc(100% - #{$navbar-width});
+	margin-left: $navbar-width;
+  color: $opfront-white;
+
+  input {
+      background-color: transparent;
+      color: inherit;
+      outline: none;
+      max-width: 200px;
+      border: $input-border-grey;
+      text-transform: none;
```

Comment:
You will probably need to add a ```filled``` class to your footer inputs so when they are filled they don't turn black as they do in the contact section. Something like:
```
&.filled {
    border-color: blue;
}
```